packages/tldraw/src/test/resizing.test.ts

```diff
@@ -3563,6 +3563,9 @@ describe('editor.resizeNoteShape', () => {
 		expect(editor.getShape(noteBId)).toMatchObject({ x: 0, y: 0, props: { scale: 2.1 } }) // but scaled!
 
 		expect(editor.getShapePageBounds(noteBId)).toMatchObject({ x: 0, y: 0, w: 420, h: 420 })
+
+		// for the sake of future tests, set it back to normal
+		NoteShapeUtil.options.resizeMode = 'none'
```

Comment:
this might cause errors in other tests if this test fails. can we use a `berfore`/`after` instead?